cts/cts-exec.in

```diff
@@ -1225,6 +1225,9 @@ def main(argv):
     opts = TestOptions()
     opts.build_options(argv)
 
+    if opts.options['show-usage']:
+        tests.show_usage()
```

Comment:
tests hasn't been created yet at this point. It is a good idea to do it before creating the temporary directory though, so maybe show_usage() should be taken out of the object and just be a regular function

we also need to exit after calling show_usage() (and after print_list() as well)

Reply:
Oops, I had accidentally tested the wrong code before I pushed... The next commit should fix it:

```
======= FINAL RESULTS ==========

--- FAILURE RESULTS:
    None

--- TOTALS
    Pass:53
    Fail:0
```